satellite/console/consoleweb/server.go

```diff
@@ -247,107 +248,107 @@ func (s *Server) createRegistrationTokenHandler(w http.ResponseWriter, req *http
 }
 
 // accountActivationHandler is web app http handler function
-func (s *Server) accountActivationHandler(w http.ResponseWriter, req *http.Request) {
-	ctx := req.Context()
+func (server *Server) accountActivationHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
-	activationToken := req.URL.Query().Get("token")
+	activationToken := r.URL.Query().Get("token")
 
-	err := s.service.ActivateAccount(ctx, activationToken)
+	err := server.service.ActivateAccount(ctx, activationToken)
 	if err != nil {
-		s.log.Error("activation: failed to activate account",
+		server.log.Error("activation: failed to activate account",
 			zap.String("token", activationToken),
 			zap.Error(err))
 
-		s.serveError(w, req)
+		server.serveError(w, r)
 		return
 	}
 
-	http.ServeFile(w, req, filepath.Join(s.config.StaticDir, "static", "activation", "success.html"))
+	http.ServeFile(w, r, filepath.Join(server.config.StaticDir, "static", "activation", "success.html"))
```

Comment:
Use the http.FileSystem created in constructor rather than `http.ServeFile`

Reply:
Good idea yeah.